### PR TITLE
Issue #13213: Removed '//ok' from InputRequireThis15Extensions along with its suppressions

### PR DESCRIPTION
**Pull request by raghuramala54**
Issue #13213: Removed '//ok' from InputRequireThis15Extensions along with its suppressions

### CODE REVIEW

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -95,8 +95,6 @@
             files="checks[\\/]coding[\\/]packagedeclaration[\\/]InputPackageDeclarationPlain.java"/>
   <suppress id="UnnecessaryOkComment"
             files="checks[\\/]coding[\\/]packagedeclaration[\\/]InputPackageDeclarationWithCommentOnly.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]coding[\\/]requirethis[\\/]InputRequireThis15Extensions.java"/>
   <suppress id="UnnecessaryOkComment"
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:98]: Everything lgtm....

@raghuramala54 , 
it is not recommended to close and open new PR. All changes can be made through git cmd on existing PR itself  ;)

**Review by MANISH-K-07 — COMMENTED** [review]
The CI are all green. Please wait for approval from maintainers now..

File: config/checkstyle-input-suppressions.xml (hunk 1 above)
Comment (raghuramala54) [reply config/checkstyle-input-suppressions.xml:98]: oh thank you for the feedback @MANISH-K-07 , felt overwhelming in the beginning. sorry abt that

**Review by nrmancuso — APPROVED** [review]
Ok to merge as CI passes, @raghuramala54 thanks for your contribution!

**Review by romani — APPROVED** [review]

**Pull request merged**